spec/config_spec.rb

```diff
@@ -3,31 +3,47 @@
 require 'spec_helper'
 
 RSpec.describe AppleSignIn do
-  let(:apple_aud) { '111111' }
+  let(:apple_client_id) { '111111' }
+  let(:apple_private_key) { '111111' }
+  let(:apple_key_id) { '111111' }
+  let(:apple_team_id) { '111111' }
+  let(:redirect_uri) { '111111' }
```

Comment:
A possible error in the Config class might be that it assigns the value to the wrong field.
I would use different values to catch that error in the test

Reply:
Agree, something like this
```suggestion
  let(:apple_client_id) { 'mocked_client_id' }
  let(:apple_private_key) { 'mocked_private_key' }
  let(:apple_key_id) { 'mocked_key_id' }
  let(:apple_team_id) { 'mocked_team_id' }
  let(:redirect_uri) { 'https://example.com/redirect_uri' }
```